src/com/google/foam/demos/olympics/Controller.js

```diff
@@ -30,8 +30,13 @@ foam.CLASS({
     'foam.u2.TableView'
   ],
 
```

Comment:
The package name of this class should be updated to reflect its new home in come.google.foam.demos